aws-rds/README.md

```diff
@@ -28,99 +28,172 @@ To access this integration, [connect to CloudWatch](https://github.com/signalfx/
-
-2. Finish up configuration
-    - For Execution role, select `Choose an existing role` and select a role with the required permissions (see Basic Information, above).
-    - Change the default 3 second timeout to 5 seconds.
-
-##### 4. Add a trigger
-
-On the Triggers tab, click Add Trigger.
-
-![](./img/trigger-before.png)
-
-Select CloudWatch Logs from the list. A number of additional options will appear. Select `RDSOSMetrics` under 'Log Group', and select an appropriate 'Filter Name'. Leave 'Filter Pattern' empty.
-
-We recommend that you deselect the 'Enable trigger' option so that you can review your settings before activating your Lambda. You can enable the trigger from the Triggers tab at any time.
-
-![](./img/trigger-after.png)
-
-When you are satisfied with your configuration settings, enable the trigger, and you're good to go! Your metrics will shortly be on their way to SignalFx, and built-in dashboards for RDS Enhanced Monitoring will appear on the Dashboards page in SignalFx.
-
-###### Optional: Specify a subset of available metrics to send to SignalFx
-To specify that SignalFx should collect only a subset of the metrics available from Enhanced Monitoring, create an environment variable to store the desired list. You can either include only the specified groups, or exclude all but the specified groups.
-
-The key for this environment variable is `groups` for the inclusive group or `groups_out` for the exclusive group. The list should simply be the names of the groups, spelled exactly as they are in the documentation (including capitals), separated by a single space. If you create both lists, only the `groups` list will be used.
-
-E.g.
-`cpuUtilization diskIO memory OSprocesses`
-
-The complete list of available metrics can be found <a target="_blank" href="http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.OS.html">here</a>.
-
-Available Metric Groups (all but SQL Server):
-
-- `cpuUtilization`
-- `diskIO`
-- `fileSys`
-- `loadAverageMinute`
-- `memory`
-- `network`
-- `OSprocesses`
-- `RDSprocesses`
-- `swap`
-- `tasks`
-
-Available Metric Groups (SQL Server only):
-
-- `cpuUtilization`
-- `disks`
-- `memory`
-- `network`
-- `OSprocesses`
-- `RDSprocesses`
-- `system`
-
-NOTE: While `OSprocesses` and `RDSprocesses` are not listed groups in the AWS documentation, they are metric groups that we provide.
+The package will be named `enhanced_rds.zip`. This will be the file to upload
+for the Lambda.
+
+##### 4. Create and configure the Lambda function
+From the Lambda creation screen, make sure you have selected
+`Build from scratch`. Select a name for your function. For `Runtime` select
+`Python2.7`. For the execution role, either select the role you wish to use or
+select `Create from Template` and add KMS decrypt permissions if need be. You
+will also need to choose a name for the role.
+
+For subsequent tabs, follow the instructions below. 
```

Comment:
Again, what does "below" actually point to? Is it the Designer heading? Can you be more specific? Do they have to do what it says for each of the headings?